scripts/manifests-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -euox pipefail
+
+SRCROOT="$( CDPATH='' cd -- "$(dirname "$0")/.." && pwd -P )"
+AUTOGENMSG="# This is an auto-generated file. DO NOT EDIT"
+
+KUSTOMIZE="${1:-}"
+if [ -z "$KUSTOMIZE" ]; then
+    echo "Path to kustomize not provided"
+    exit 1
+fi
+
+IMAGE_TAG="${2:-}"
+if [ -z "$IMAGE_TAG" ]; then
+    echo "Image tag not provided"
+    exit 1
+fi
+
+IMAGE_NAMESPACE="${IMAGE_NAMESPACE:-quay.io/argoprojlabs}"
+IMAGE_FULLNAME="$IMAGE_NAMESPACE/argocd-ephemeral-access:$IMAGE_TAG"
```

Comment:
Should we redefine defaults here or trust the makefile to send the correct args? 
Makefile already contains 
```
# docker image publishing options
IMAGE_NAMESPACE?=argoproj-labs
IMAGE_NAME=${IMAGE_NAMESPACE}/argocd-ephemeral-access
```

Reply:
@agaudreault I removed the `IMAGE_NAMESPACE` from the script to avoid conflict. However it was working as expected. 
BTW, you can validate the manifest release with: `make goreleaser-build-local` and inspect the dist/install.yaml file